scripts/in_container/entrypoint_ci.sh

```diff
@@ -98,13 +101,51 @@ if [[ -z ${INSTALL_AIRFLOW_VERSION=} ]]; then
     mkdir -p "${AIRFLOW_SOURCES}"/logs/
     mkdir -p "${AIRFLOW_SOURCES}"/tmp/
     export PYTHONPATH=${AIRFLOW_SOURCES}
+elif [[ ${INSTALL_AIRFLOW_VERSION} == "none"  ]]; then
+    echo
+    echo "Skip installing airflow - only install wheel packages that are present locally"
+    echo
+    uninstall_airflow_and_providers
+elif [[ ${INSTALL_AIRFLOW_VERSION} == "wheel"  ]]; then
+    echo
+    echo "Install airflow from wheel package with [all] extras but uninstalling providers."
+    echo
+    uninstall_airflow_and_providers
+    install_airflow_from_wheel "[all]"
+    uninstall_providers
 else
-    # Installs released airflow version without adding more extras
-    install_released_airflow_version  "" "${INSTALL_AIRFLOW_VERSION}"
+    echo
+    echo "Install airflow from PyPI including [all] extras"
+    echo
+    install_released_airflow_version "${INSTALL_AIRFLOW_VERSION}" "[all]"
 fi
-
-if [[ ${INSTALL_WHEELS=} == "true" ]]; then
-  pip install /dist/*.whl || true
+if [[ ${INSTALL_PACKAGES_FROM_DIST=} == "true" ]]; then
+    echo
+    echo "Install all packages from dist folder"
+    if [[ ${INSTALL_AIRFLOW_VERSION} == "wheel" ]]; then
+        echo "(except apache-airflow)"
+    fi
```

Comment:
Why don't we install airflow from the sdist?

Reply:
Because it takes much longer. That's the only reason. From observations, it takes usually 2-4 times longer to install an sdist package than .whl, And this test already runs for quite a few minutes, so I wanted to reduce the strain on CI

Do you think @ashb we should? Do you expect any problems with sdist comparing to wheel? I can easily change it to install from sdist additionally to wheels, but it will make our tests quite a bit longer, so unless there is a good reason we suspect something wrong we should not do it. 

The only reason I am installing sdist for providers is that the setup.py are generated there and they were wrong initially, but i was considering to drop that as well, as I do not expect any more problems there - sdist should behave same as .wh  if setup.py is there

Reply:
BTW. I think we'd loose 4/5 times more than from optimising the extra step in build workflow. to be honest if we install .sdist. Do you think it's worth it ?

Reply:
Haven't got an opinion (yet, can form one if needed) - just wondering why this was done

Reply:
I looked at it - and the way I implemented it is rather easy to adjust - I could run preparing packages and build images jobs in matrix strategy witth PACKAGE_FORMAT ["wheels", "sdist" ] - I could get it done rather quickly, but I am still not sure if it gives us anything more than installing .whl - > if you can think of a reason why this might be useful, I am happy to add it.

Reply:
Is it good to go @ashb as is ?